### PR TITLE
Create JSONWriter Module

### PR DESCRIPTION
**Pull request by paritoshgpt1**

### CODE REVIEW

File: bin/boom [hunk 1]
```diff
@@ -145,6 +156,16 @@ if __name__ == '__main__':
         # conf['pipeline']['rabbitmq_host'] = rabbitmq.attrs['NetworkSettings']['IPAddress']
         conf['pipeline']['rabbitmq_host'] = '127.0.0.1' # Using host mode for the network
 
+    elif conf['pipeline']['mode'] == 'aws':
+        tf = Terraform(working_dir='../../boom/infra')
```
Comment (liboyue) [inline bin/boom]: This should be an arg that users can set when calling BOOM. You can add a new option to the binary.

Comment (paritoshgpt1) [reply bin/boom]: The way I have designed it is to keep it abstracted from the user. The user need not worry how everything is happening and that is the reason why currently I am not taking this as an input from the user.
Also, my bad, this should have not been in this branch. This should come with the aws code

Comment (liboyue) [reply bin/boom]: User does not need to work in the same directory, so that the relative path would definitely fail.

Comment (paritoshgpt1) [reply bin/boom]: Ok thanks. This has already been taken care of in the aws code branch. I removed it from this pull request.

File: bin/boom [hunk 2]
```diff
@@ -101,6 +106,11 @@ def start_container(client, cmd, name, volumes=None, network_mode='host'):
 
         return None
 
+def create_server(tf):
```
Comment (liboyue) [inline bin/boom]: Could you document this function please?

File: boom/job.py [hunk 3]
```diff
@@ -7,10 +7,13 @@
 class Job(object):
     """The information object class."""
 
-    def __init__(self, job_id, input_uri, output_base, output_path, params=None,
-                 producer=None, consumer=None, processing_time=None):
+    def __init__(self, job_id, input_uri, output_base, output_path,
+                 params=None, producer=None, consumer=None,
+                 processing_time=None, config=None):
 
         ## The id of this job
```
Comment (liboyue) [inline boom/job.py]: This comment is in the wrong line.

File: bin/boom (hunk 2 above)
Comment (paritoshgpt1) [reply bin/boom]: Removing it.

File: boom/job.py (hunk 3 above)
Comment (paritoshgpt1) [reply boom/job.py]: Fixing this.

**Pull request merged**